pages/landingpage.js

```diff
@@ -18,10 +63,26 @@ function  LandingPage({ user } = props) {
                 <br></br>
                 <br></br>
                 <br></br>
+                <form action="#" method="POST" onSubmit={(e) => addUser(e)}>
+                    <label>Add user content here: </label> 
+                    <input type="text" className='user-content'
+                     value={userContent} onChange={handleUserContent} 
+                    />
+                    <br /> 
+                    <br /> 
+                    <button type="submit">Submit</button> 
+                </form>
```

Comment:
No need for this here, it will be something managed in the configurations section that Chris is currently working on